chain/internal/tendermint/tendermint_node.go

```diff
@@ -137,25 +138,18 @@ func (tn *TendermintNode) PrivValKeyFilePath() string {
 	return filepath.Join(tn.Dir(), "config", "priv_validator_key.json")
 }
 
-func (tn *TendermintNode) TMConfigPath() string {
-	return filepath.Join(tn.Dir(), "config", "config.toml")
-}
-
-func (tn *TendermintNode) TMConfigPathContainer() string {
-	return filepath.Join(tn.HomeDir(), "config", "config.toml")
-}
-
 // Bind returns the home folder bind point for running the node
 func (tn *TendermintNode) Bind() []string {
 	return []string{fmt.Sprintf("%s:%s", tn.Dir(), tn.HomeDir())}
 }
 
 func (tn *TendermintNode) HomeDir() string {
-	return filepath.Join("/tmp", tn.Chain.Config().Name)
+	return path.Join("/tmp", tn.Chain.Config().Name)
```

Comment:
Just curious: Why the change from `filepath` to `path`? Docker volumes always `/`?

Reply:
Correct, within the Docker volumes, we can rely on forward slashes for paths.